src/uci.cpp

```
@@ -0,0 +1,425 @@
+    return this->value;
+}
+
+void UCIOption::setValue(std::string& value) {
+    this->value = value;
+}
+
+std::string UCIOption::getDefaultValue() {
+    return this->defaultValue;
+}
+
+void UCIOption::setDefaultValue(std::string value) {
```

cppcheck

[performance] Function parameter 'value' should be passed by const reference.

(passedByValue)